src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,14 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables) {
+        StringBuilder builder = new StringBuilder();
```

Comment:
Good job on using StringBuilder for the concatenation. Another option can be RingJoiner.

Reply:
StringJoiner*